content-src/components/DiscoveryStreamComponents/List/_List.scss

```diff
@@ -14,6 +14,19 @@ $item-line-height: 20;
   margin-block-end: 8px;
 }
 
+// XXX this is gross, and attaches the bottom-border to the item above.
+// Ideally, we'd attach the top-border to the item that needs it.
+// Unfortunately the border needs to go _above_ the row gap as currently
+// set up, which means that some refactoring will be required to do this.
+@mixin bottom-border-except-last-grid-row($columns) {
```

Comment:
This would be more accurately named something along of "except last item in each column" as opposed to "last row"

Reply:
Right now, the patch I have up does exactly what the name suggests.  Since it sounds like we're going to want to file another bug and change that behavior, I suggest we change the name then.

Reply:
Actually, the above comment is wrong.  But I vote we leave things alone and I'll file that bug where we can address the behavioral problem.